src/sass/variables.scss

```diff
@@ -3,6 +3,11 @@ Provides SASS variables with Myworkout design details.
 These values can be updated by client apps when appropriate.
  */
 
+$mw-mobile-breakpoint: 980px;
```

Comment:
Not sure if it actually makes sense to have this as a reusable variable? Like, what does the value actually represent and when/where should it be used?

Reply:
It's one of the variables that we have repeatead in the API and in the Webapp.

Reply:
I know, but I still have the question "what does the it actually represent and when/where should it be used"?

In the Webapp the variable represents when it should go from "desktop mode" with a left-hand menu to "mobile mode" where it replaces the left-hand menu with a hamburger menu button. 
While in the API it's only used in a very specific use-case, and we use a different breakpoint to decide when we should go from "desktop mode" where the login form is centered to "mobile mode" where the login form is full-width.

In other words: Yes, We have the same variable in both places, but it seems to be used for different things. So does it make sense to have it re-usable then? If it changes in 1 place, should it also change in the other?

Reply:
> If it changes in 1 place, should it also change in the other?
Probably not 👍🏻. Also, why would we change it? When mobile devices are wider? 🤣 

Removing it 😄